build.fsx

```diff
@@ -329,34 +329,32 @@ Target "BuildPackage" DoNothing
 
 
 // --------------------------------------------------------------------------------------
-// .NET CLI and .NET Core
+// .NET Core SDK and .NET Core
 
 let assertExitCodeZero x = if x = 0 then () else failwithf "Command failed with exit code %i" x
-let netcoreFW = "netstandard1.5"
 
-Target "DotnetCliBuild" (fun _ ->
+Target "Build.NetCore" (fun _ ->
     Shell.Exec("dotnet", "restore") |> assertExitCodeZero
-    Shell.Exec("dotnet", sprintf "--verbose build --framework %s --configuration Release" netcoreFW, "src/Chessie") |> assertExitCodeZero
+    Shell.Exec("dotnet", "--verbose pack --configuration Release", "src/Chessie") |> assertExitCodeZero
```

Comment:
because my goal/flow is to **just** add netcore to an existing package:

1. to create a package with .net core (`dotnet pack`)
2. merge the package with the one built from fake (`dotnet mergenupkg`)

the `dotnet pack` does `dotnet build` if needed, one step less